faq/osgi/README.md

```diff
@@ -1,7 +1,3 @@
-```eval_rst
```

Comment:
Please squash this commit with the first commit in the PR. Otherwise the first commit will create conflicts anyway.